docker-compose.yml

```diff
@@ -13,6 +13,8 @@ services:
     environment:
       DATABASE_URL: postgres://postgres@db/askdarcel_development
       TEST_DATABASE_URL: postgres://postgres@db/askdarcel_test
+      ALGOLIA_APPLICATION_ID:
+      ALGOLIA_API_KEY:
```

Comment:
These keys are required now, so you'll have to sign up for Algolia and create a new 'application', then copy/paste the values into your local `.env` file.